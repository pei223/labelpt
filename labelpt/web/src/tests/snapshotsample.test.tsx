import React from 'react'
import renderer from 'react-test-renderer'
import LabelItem from '../components/atoms/label_item'
import Label from '../domain/label'

it('Snapshot sample', () => {
  const tree = renderer.create(
    <LabelItem
      width="100px"
      labelInfo={new Label(1, 'test')}
      selected={false}
      onClick={() => {}}
    />
  )

  expect(tree).toMatchSnapshot()
})
