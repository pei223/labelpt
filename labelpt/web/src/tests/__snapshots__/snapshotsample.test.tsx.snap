// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot sample 1`] = `
<button
  className="MuiButtonBase-root MuiButton-root MuiButton-outlined MuiButton-outlinedSizeSmall MuiButton-sizeSmall"
  disabled={false}
  onBlur={[Function]}
  onClick={[Function]}
  onDragLeave={[Function]}
  onFocus={[Function]}
  onKeyDown={[Function]}
  onKeyUp={[Function]}
  onMouseDown={[Function]}
  onMouseLeave={[Function]}
  onMouseUp={[Function]}
  onTouchEnd={[Function]}
  onTouchMove={[Function]}
  onTouchStart={[Function]}
  style={
    Object {
      "backgroundColor": "white",
      "borderColor": "rgba(168, 40, 40, 0.9)",
      "borderWidth": "2px",
      "color": "rgba(168, 40, 40, 0.9)",
      "textTransform": "none",
      "width": "100px",
    }
  }
  tabIndex={0}
  type="button"
>
  <span
    className="MuiButton-label"
  >
    1
    : 
    test
  </span>
</button>
`;
